### ISSUE TITLE
Sourcemap file not found, although it exists in the file system

### ISSUE THREAD

**Issue by Ollisteka** [getsentry/sentry-cli#1825]
### Environment
@sentry/cli `v2.20.0` and `2.21.3`
node `v18.12.1`
Windows 10

### Steps to Reproduce
Running the following command:
```
sentry-cli --url=<CUSTOM_URL> --auth-token=<AUTH_TOKEN> --log-level=debug sourcemaps inject --org=<ORG> --project=<PROJECT>  C:\cabinet-git\Cabinet\Cabinet\RA\Static\Bundles\js
```

### Expected/Actual Result
Expected `js.map` files to have a `debug_id` property, but there is none.

Minified `.js` file does get injected.

### Logs

Debug logs telling me that `js.map` files are not found, but they exist

![image](https://github.com/getsentry/sentry-cli/assets/27495566/f95ee9fd-7aca-4373-bfdc-c7d3cfcab563)

DebugId in minified file:
![image](https://github.com/getsentry/sentry-cli/assets/27495566/3ff29e66-3d65-4bb3-9e26-027b41b2cc6a)

**Comment by loewenheim** [comment]
Hi, thank you for the report. That definitely looks very strange. Would it be possible for you to send me the folder containing the JS files and sourcemaps at sebastian.zivota@sentry.io?

Side note: for `sourcemaps inject` you don't need to pass a URL, org, project, or auth token since it's an entirely offline operation.

**Comment by loewenheim** [comment]
Thank you for the files. On my Macbook I can't reproduce this; I'll try on my Windows computer.

**Comment by loewenheim** [comment]
Sorry, I can't make it happen in Windows either. After deleting the debug ids from the JS files:
```
.\sentry-cli-Windows-x86_64.exe sourcemaps inject --dry-run js
> Searching js
> Found 42 files
> Analyzing 42 sources
> Analyzing completed in 0.017s
> Injecting debug ids

Source Map Debug ID Injection Report
  Modified: The following source files have been modified to have debug ids
    2c89d023-d9b5-5427-ba56-9aff97dfde4b - js\common-components-af6a060d61decd158eef.js
    894ba2b4-8fa0-58c7-846c-dde77adb99ac - js\core-6ef46a68149f93679d30.js
    1e46cffb-32ed-504e-94ce-c38f56b4a6ac - js\download-mobile-signature-app-page-19aa0eb596cd85171d3c.js
    55742c2b-2898-5f6f-bcba-047cd2b08eaf - js\download-mydss-app-page-113e16da31585182f934.js
    18029391-b82a-5997-88b6-c58ea274fefd - js\dss-activation-page-269048e9a4bf27a6ac4d.js
    62c87fa3-fbef-5495-9676-a51d97738364 - js\dss-info-page-7d0b816803e546512baf.js
    770d9b80-0dd7-51e0-9071-1143b5404235 - js\dynamic\file-type-3f690b8a79e321dfe05e.js
    85b400e1-c30c-54db-a101-0a939fdc1855 - js\dynamic\heic2any-a9389d8d65c51c9417ed.js
    8ca5b9da-00da-5cbd-881b-552b2d6376ca - js\enter-8bdba1d99f9677e0828e.js
    4141e232-6a1e-5104-a9cb-58909e191dc3 - js\error-handler-c0625f6cddb43d340a22.js
    e56ae561-8b38-59a4-bddf-720da0d1e12b - js\error-page-30ac5c57dd0d04231749.js
    2af47e12-2cbb-5465-ba32-14cd46859c65 - js\full-cert-info-page-92213072a6f08463237c.js
    d1c764d2-449f-5b22-8c3e-d84fb70f105a - js\invalid-browser-page-9bdfde73090c72e04a0e.js
    a903ce1a-bf1e-5345-aa80-c4bdd4544a4a - js\list-page-9ba3609f10272a7a2d16.js
    63e903f1-ed16-53b7-8b00-af4dc93a34ca - js\login-by-token-page-71540393ca1ab01e1c10.js
    254758a5-ce00-5d9c-b298-2f22f9a4728c - js\modules-7f427ce990706bb34865.js
    80a0a1c5-1e3c-57cd-aad4-39961eee95ed - js\readonly-form-page-98a925a3cb590e689b8b.js
    c521077a-1fa1-5da7-9472-4a45cc22337d - js\runtime-9bc823e064578d15043b.js
  Modified: The following sourcemap files have been modified to have debug ids
    2c89d023-d9b5-5427-ba56-9aff97dfde4b - js\common-components-af6a060d61decd158eef.js.map
    894ba2b4-8fa0-58c7-846c-dde77adb99ac - js\core-6ef46a68149f93679d30.js.map
    1e46cffb-32ed-504e-94ce-c38f56b4a6ac - js\download-mobile-signature-app-page-19aa0eb596cd85171d3c.js.map
    55742c2b-2898-5f6f-bcba-047cd2b08eaf - js\download-mydss-app-page-113e16da31585182f934.js.map
    18029391-b82a-5997-88b6-c58ea274fefd - js\dss-activation-page-269048e9a4bf27a6ac4d.js.map
    62c87fa3-fbef-5495-9676-a51d97738364 - js\dss-info-page-7d0b816803e546512baf.js.map
    770d9b80-0dd7-51e0-9071-1143b5404235 - js\dynamic\file-type-3f690b8a79e321dfe05e.js.map
    85b400e1-c30c-54db-a101-0a939fdc1855 - js\dynamic\heic2any-a9389d8d65c51c9417ed.js.map
    8ca5b9da-00da-5cbd-881b-552b2d6376ca - js\enter-8bdba1d99f9677e0828e.js.map
    4141e232-6a1e-5104-a9cb-58909e191dc3 - js\error-handler-c0625f6cddb43d340a22.js.map
    e56ae561-8b38-59a4-bddf-720da0d1e12b - js\error-page-30ac5c57dd0d04231749.js.map
    2af47e12-2cbb-5465-ba32-14cd46859c65 - js\full-cert-info-page-92213072a6f08463237c.js.map
    d1c764d2-449f-5b22-8c3e-d84fb70f105a - js\invalid-browser-page-9bdfde73090c72e04a0e.js.map
    a903ce1a-bf1e-5345-aa80-c4bdd4544a4a - js\list-page-9ba3609f10272a7a2d16.js.map
    63e903f1-ed16-53b7-8b00-af4dc93a34ca - js\login-by-token-page-71540393ca1ab01e1c10.js.map
    254758a5-ce00-5d9c-b298-2f22f9a4728c - js\modules-7f427ce990706bb34865.js.map
    80a0a1c5-1e3c-57cd-aad4-39961eee95ed - js\readonly-form-page-98a925a3cb590e689b8b.js.map
    c521077a-1fa1-5da7-9472-4a45cc22337d - js\runtime-9bc823e064578d15043b.js.map
```

**Comment by Ollisteka** [comment]
Oh, I think I found the "fix"! I was executing the script from another folder, not the one with the bundles

So this code does not find sourcemap:
```
C:\cabinet-git\Cabinet\Cabinet> sentry-cli --log-level=debug sourcemaps inject C:\cabinet-git\Cabinet\Cabinet\RA\Static\Bundles\js
```
But this does:
```
C:\cabinet-git\Cabinet\Cabinet\RA\Static\Bundles\js> sentry-cli --log-level=debug sourcemaps inject ./
```

**Comment by Ollisteka** [comment]
I managed to configure `sentryWebpackPlugin` so it would upload the sourcemaps, it works fine, no problems with path. So I guess this issue can be closed... But on the other hand, even with the webpack plugin, stacktrace sourcemapping does not work! I went through [the troubleshooting](https://blog.sentry.io/how-to-fix-source-map-upload-errors/), fixed every error I could find, but still no luck. Should I create a separate issue, or can we continue the discussion in this one?
